### ISSUE TITLE
Update the deploy Dockerfile too

### ISSUE THREAD

**Issue by lemeurherve** [jenkins-infra/plugin-site-api#107]
Follow-up of https://github.com/jenkins-infra/plugin-site-api/pull/105

The Dockerfile at https://github.com/jenkins-infra/plugin-site-api/blob/4c28d164ac79c003865d6d3639da27a228d50c6f/deploy/Dockerfile#L1 should use the same image as https://github.com/jenkins-infra/plugin-site-api/blob/4c28d164ac79c003865d6d3639da27a228d50c6f/Dockerfile#L1

**Comment by sridamul** [comment]
Hey, @lemeurherve!
Is there any reason to stick to jdk8?
else
Merging PR #124 will resolve this issue right?

**Comment by halkeye** [comment]
I would assume this super old project doesn't run in newer java

**Comment by halkeye** [comment]
@lemeurherve I would argue a ticket with no details whats so ever is not a good first issue

**Comment by halkeye** [comment]
It happens. I fight that fight a lot at work so could have been more polite sorry.

So the request is copy 
https://github.com/jenkins-infra/plugin-site-api/blob/master/Dockerfile#L1
To
https://github.com/jenkins-infra/plugin-site-api/blob/master/deploy/Dockerfile then confirm it works?

**Comment by lemeurherve** [comment]
Yes exactly, thanks!

Issue body updated.

**Comment by SalmanDeveloperz** [comment]
Hey @lemeurherve , @halkeye , I’d like to pick this up!

From what I understand, the task is to update [`deploy/Dockerfile`](https://github.com/jenkins-infra/plugin-site-api/blob/master/deploy/Dockerfile) so it uses the same base image `(jetty:9-jdk8-eclipse-temurin`) as the main Dockerfile, and then check that the deploy image still builds and runs fine.

Or is there more to it (like java compat checks we should keep in mind)? thx!

**Comment by lemeurherve** [comment]
That should be sufficient, to be properly tested locally to be sure though.

**Comment by lemeurherve** [comment]
In retrospect and since we had recent issues around this service, I'd prefer not touching this repo for now, at least not until mid March/April (low bandwidth, and this change is not urgent at all).

I also may take care of this change myself as there are other things I've noticed recently that might need some update too.

**Comment by SalmanDeveloperz** [comment]
Make sense, thanks for the heads‑up. I’ll leave this one alone for now. if there are other infra issues/repos that could use a hand in the meantime, happy to jump in there.

**Comment by lemeurherve** [comment]
Thanks, I'll keep that in mind 🙂 

BTW, feel free to pass to say hello in one of our incoming infra meetings 😉 

https://www.jenkins.io/projects/infrastructure/#meetings

**Comment by dduportal** [comment]
Closing in favor of https://github.com/jenkins-infra/plugin-site-api/issues/164#issuecomment-3891913262. We have no reason to spend time on the Dockerfile of a project + service that we aim to kill

**Comment by dduportal** [comment]
In addition: https://github.com/jenkins-infra/plugin-site-api/blob/master/deploy/Dockerfile is not used in production.

We use the [`Dockerfile` at the root of the repository](https://github.com/jenkins-infra/plugin-site-api/blob/master/Dockerfile): https://github.com/jenkins-infra/plugin-site-api/blob/79564bcfebd4840cc3cdf050f496528ec938b017/Jenkinsfile#L86-L93

There are no mention of the `deploy/Dockerfile ` except an old `Makefile`: https://github.com/search?q=repo%3Ajenkins-infra%2Fplugin-site-api%20deploy&type=code
